TextToSpeech/impl/TTSSpeaker.h

```diff
@@ -121,6 +127,7 @@ class TTSSpeaker {
     void queueData(SpeechData);
     void flushQueue();
     SpeechData dequeueData();
+    PipelineType m_pipelinetype;
```

Comment:
m_pipelineType & m_pcmAudioEnabled seem to indicate similar state information and redundant.

Reply:
m_pipelineType used only on devices with offline/online switch enabled.
m_pcmAudioEnabled is used on xclass 
will refine this as part of tts code refinement

Reply:
Can't we decide if PCM audio is enabled with m_pcmAudioEnabled?